### PR TITLE
Databases/week2

### PR DESCRIPTION
**Pull request by Jul-S**

### CODE REVIEW

File: databases/week2/queries_part1.sql [hunk 1]
```diff
@@ -0,0 +1,58 @@
+-- Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+insert into
```
Comment (calindurnea) [inline databases/week2/queries_part1.sql]: Uppercase

File: databases/week2/queries_part1.sql [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  task (
+    id,
```
Comment (calindurnea) [inline databases/week2/queries_part1.sql:4]: I think this should be `user_id` 😁

File: databases/week2/school_database.sql [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    `email` varchar(255) NOT NULL,
+    `phone` varchar(255) NULL,
+    `class_id` int(10) unsigned NOT NULL,
+    CONSTRAINT `fk_class` FOREIGN KEY (`class_id`) REFERENCES `class` (`id`) ON DELETE CASCADE
+  ) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+CREATE INDEX idx_name ON student (name);
+ALTER TABLE
+  class
+ADD
+  status ENUM('not-started', 'ongoing', 'finished');
```
Comment (calindurnea) [inline databases/week2/school_database.sql:21]: Perfect 🥇

File: databases/week2/hotel_database.sql [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    `move_out` DATETIME NOT NULL
+  ) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+CREATE TABLE `status` (
+    `id` int(10) unsigned NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    `name` varchar(255) NOT NULL
+  ) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+CREATE TABLE `room` (
+    `id` int(10) unsigned NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    `type` varchar(255) NOT NULL,
+    `NumberOfGuests` int(10) unsigned NOT NULL,
```
Comment (calindurnea) [inline databases/week2/hotel_database.sql]: You could consider renaming this to `number_of_guests` for consistency

**Review by calindurnea — COMMENTED** [review]
Hello,

I left a few comments but overall its very good, including file naming and organizing! Keep up the great job 🥇 

Let me know if there is anything that I should clarify.

p.s. for creating Entity Relationship Diagrams (ERDs) you can find some free software online, i usually use: https://erdplus.com/standalone or https://app.diagrams.net/

**Comment by Jul-S** [comment]
Thank you so much for review! I`ve updated my PR with your comments

**Review by calindurnea — APPROVED** [review]

**Pull request merged**